app/plugin_test.go

```diff
@@ -48,19 +48,39 @@ func TestPluginKeyValueStore(t *testing.T) {
 	assert.Nil(t, th.App.DeletePluginKey(pluginId, "postkey"))
 	assert.Nil(t, th.App.DeletePluginKey(pluginId, "notrealkey"))
 
-	// Test ListKeys
-	assert.Nil(t, th.App.SetPluginKey(pluginId, "key2", []byte("test")))
-	hashedKey := getHashedKey("key")
 	hashedKey2 := getHashedKey("key2")
+	kv := &model.PluginKeyValue{
+		PluginId: pluginId,
+		Key:      hashedKey2,
+		Value:    []byte("test"),
+		ExpireAt: 0,
+	}
+
+	result := <-th.App.Srv.Store.Plugin().SaveOrUpdate(kv)
+	assert.Nil(t, result.Err)
+
+	// Test fetch by hashed keyname
```

Comment:
This comment and the one line line 67 seem like they should be swapped? This is testing the key, and the latter the hashed key, right?

Reply:
Also, I can't comment on earlier lines, but:
* line 39: Can you add a check that verifies that `GetPluginKey` now returns `test2`?